tests/pytest/eligibility/test_views.py

```diff
@@ -13,7 +13,7 @@
 
 from benefits.core import session
 from benefits.core.models import TransitAgency, EligibilityVerifier
-from benefits.eligibility.api import ApiError, TokenError
+from eligibility_api.api import ApiError, TokenError
```

Comment:
See my other comment related to cal-itp/eligibility-api#26 and organization within the package.